### DISCUSSION TITLE
Modify SideAdvectiveFluxIntegralForce

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#27008] [Q&A General] [answered]
Is there a way to calculate the intergral of grad_u * diffusivity * _normals[_qp] * _direction, where _direction is defined as a real vector value ?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Certainly but you ll have to make a new object.
I would create a vector postprcoessor OR a reporter to compute this

Guillaume

**Reply by TLWise** [reply]
Thank you for your continued guidance.

**Reply by TLWise** [reply]
@GiudGiud, Is there an example vector postprocessor to that incorporates the velocity vectors on a sideset?

**Reply by GiudGiud** [reply]
I dont think so.
But VPPs look a lot like PPs and what you have to do is close to the SideDiffusiveFluxIntegral PP so it s a good inspiration.
Look for a VPP that derives from a SideVPP base class as an example of a VPP that does integration on a side
